### PR TITLE
Update docugami reader to optionally return XML markup in chunks

### PR DESCRIPTION
**Pull request by tjaffri**
# Description

Optionally return XML markup in chunks from the docugami reader. This can be used to enhance RAG, using semantic cues on chunks.

## Type of Change

Please delete options that are not relevant.

- [ ] New Loader/Tool
- [x] Bug fix / Smaller change
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Added new unit/integration tests
- [x] Added new notebook (that tests end-to-end)
- [ ] I stared at the code and made sure it makes sense

# Suggested Checklist:

- [ ] I have added a library.json file if a new loader/tool was added
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] I ran `make format; make lint` to appease the lint gods

### CODE REVIEW

**Review by kenzie28 — APPROVED** [review]

File: llama_hub/docugami/base.py [hunk 1]
```diff
@@ -30,8 +33,19 @@ class DocugamiReader(BaseReader):
...
+    """Access token for API endpoint."""
+
+    max_metadata_length = DEFAULT_MAX_METADATA_LENGTH
+    """Max length of metadata values."""
+
+    min_chunk_size: int = DEFAULT_MIN_CHUNK_SIZE
+    """Threshold under which chunks are appended to next chunk to avoid over-chunking."""
+
+    include_xml_tags: bool = False
+    """Aet to true for XML tags in chunk output text."""
```
Comment (anoopshrma) [inline llama_hub/docugami/base.py:48]: Small typo here! 
Should be *Set* to true ...

**Review by anoopshrma — APPROVED** [review]

**Comment by EmanuelCampos** [comment]
@tjaffri any reason to set it as a draft?

**Comment by tjaffri** [comment]
@EmanuelCampos will publish it soon, I hope... still working on a few improvements sorry!

**Comment by anoopshrma** [comment]
Hey @tjaffri , This PR has been in draft and I see that you can open a new PR for Docugami. Are they both for the same work?
If so, Can we go ahead and close this PR?

**Comment by tjaffri** [comment]
@anoopshrma closed, sorry to leave this hanging! Yes we have a new PR coming shortly with the changes in this one plus other improvements…. Will tag you there.

**Comment by anoopshrma** [comment]
Hey no worries!, Sure 😄

**Pull request closed without merging**